recipe/meta.yaml

```
@@ -24,19 +27,20 @@ requirements:
     - setuptools_scm
     - wheel
     - cython >=0.29.24
-    - numpy {{ numpy }}
+    - numpy 2.0 # [py<313]
+    - numpy 2.1 # [py==313]
   run:
     - python
-    - {{ pin_compatible('numpy') }}
-    - shapely >=1.7
-    - packaging >=20
-    - pyshp >=2.1
-    - matplotlib-base >=3.4
+    - numpy >=1.23
+    - shapely >=1.8
+    - packaging >=21
+    - pyshp >=2.3
+    - matplotlib-base >=3.6
     - pyproj >=3.1.0
```

```suggestion
    - pyproj >=3.3.1
```
https://github.com/SciTools/cartopy/blob/v0.24.1/pyproject.toml#L53